databases-week3/anjali

anjali744 opened the pull request.

anjali744 committed:
complete week3 homework

anjali744 committed:
complete js1-week4

anjali744 committed:
Week4-reviewed code corrected

anjali744 committed:
add homework.sql file week1

anjali744 committed:
complete week3 homework

MercedesUbeira reviewed (approved):
Hey Anjali! Really nice work here! ❤️ 

Some small comments: 

•You forgot to checkout from main for this branch, and that is why I see this homework mixed with other ones. 

•A suggestion for improving this code would be to have NOT NULL for some of the values on the tables, to make sure that they can't be empty.

•Also, perhaps it would be nice to have DATETIME instead of DATE? so you also get the time.

anjali744 committed:
Correction-Adding NOT NULL & replace DATE by DATETIME

anjali744 commented:
Hi Mechii!

Thank you for reviewing my homework. I corrected all the errors you pointed out, and I hope it's now up to standard. If you find any remaining issues, please let me know so I can fix them promptly.

I apologize for forgetting to checkout from the main branch. I have corrected that oversight. Additionally, I've taken care of the issue with the node modules.

Thank you for your patience and guidance.
